boards/arm64/xenvm/xenvm-qemu_defconfig

```diff
@@ -26,7 +26,6 @@ CONFIG_LOG=y
 CONFIG_LOG_MODE_MINIMAL=n
 
 CONFIG_TEST=y
-#CONFIG_PRINTK_SYNC=y
```

Comment:
Looks like spurious change. It is right, but at least it should be in separate commit.